Sources/HAP/Base/Generated.swift

```diff
@@ -314,6 +316,12 @@ public class Enums {
 		case auto = 1
 	}
 
+	public enum CurrentVisibilityState: UInt8, CharacteristicValueType {
+		case shown = 0
+		case hidden = 1
+        case state2 = 2
```

Comment:
Did you add these by hand?

Reply:
The indentation is off and uses spaces (however I would rather use spaces throughout).

Reply:
I did add these two lines manually to `Generated.swift` for simplicity, but I also added the change to `Inspector.swift` so the same result would be generated in this way automatically next time.